pkgs/development/python-modules/django-cacheops/default.nix

```diff
@@ -1,27 +1,41 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, fetchpatch
 , django
 , funcy
 , redis
-, pytest-django
+, six
 , pytestCheckHook
+, pytest-django
+, mock
+, dill
+, jinja2
+, before_after
 , pythonOlder
-, six
+, pkgs
 }:
 
 buildPythonPackage rec {
   pname = "django-cacheops";
-  version = "6.2";
+  version = "7.0.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-zHP9ChwUeZJT/yCopFeRo8jSgCIXswHnDPoIroGeQ48=";
+    hash = "sha256-Ed3qh90DlWiXikCD2JyJ37hm6lWnpI+2haaPwZiotlA=";
   };
 
+  patches = [
+    # support for funcy >2
+    (fetchpatch {
+      url = "https://github.com/Suor/django-cacheops/commit/b778177a1cbc776ace79ec234c25bdca8aa181ab.diff";
```

Comment:
Since this only changes the version pin, we prefer using substituteInPlace or sed to reduce the change of apply conflicts

Reply:
Or pythonRelaxDepsHook?

Reply:
If that works here, sure. I haven't used that to much yet, so I sometimes forget it exists